### PR TITLE
Update tutorial google sheet

### PR DESCRIPTION
**Pull request by damianooldoni**
As found by @LienReyserhove #190 the tutorial about reading google sheets in R has to be updated. As mentioned by @florisvdh in same issue, it is not really very useful to write a tutorial about a package which is extensively documented.

This PR changes drastically the [tutorial](https://inbo.github.io/tutorials/tutorials/r_google_sheets/) as it is sensibly reduced.

Please, get a look and happy review. Everyone can review of course, just add yourself as reviewer! Thanks.

### CODE REVIEW

File: content/tutorials/r_google_sheets/index.md [hunk 1]
```diff
@@ -1,134 +1,27 @@
...
-library(ggplot2)
-```
+## `googlesheets4`
 
-Setup
------
+The R package [googlesheets4](https://googlesheets4.tidyverse.org/) provides the functionality to read and write data from a Google sheet.
 
-The R package [googlesheets](https://github.com/jennybc/googlesheets/blob/master/README.md) provides the functionality to retrieve data from a google sheet. Once the registration as a user is done and the permissions are granted, it enables to read and write to google sheets. Initiation of the authentification can be done with the `gs_auth` command.
+This package is very well documented, so we have just to provide you with the right links:
```
Comment (florisvdh) [inline content/tutorials/r_google_sheets/index.md:14]:
```suggestion
This package is very well documented, so we just have to provide you with the right links:
```

File: content/tutorials/r_google_sheets/index.md [hunk 2]
```diff
@@ -1,134 +1,27 @@
...
-  # first get the token, but not caching it in a local file
-  token <- gs_auth(cache = FALSE)
-  # save the token into a file in the current directory
-  saveRDS(token, file = "googlesheets_token.rds")
-}
-```
+## `googlesheets4` vs `googlesheet`
 
-Make sure the token is savely stored within your project folder without sharing it or putting it into the version control history. If you need more power (e.g. necessity of integration services such as Travis CI), check the encryption options in [this manual](https://rawgit.com/jennybc/googlesheets/master/vignettes/managing-auth-tokens.html).
+The package `googlesheets4` is the follower of [`googlesheets`](https://github.com/jennybc/googlesheets/blob/master/README.md) which doesn't work anymore as it is built on the outdated Sheets v3 API. If you have still code using this package, you will get this error:
```
Comment (florisvdh) [inline content/tutorials/r_google_sheets/index.md:23]:
```suggestion
The package `googlesheets4` is the follower of [`googlesheets`](https://github.com/jennybc/googlesheets/blob/master/README.md) which doesn't work anymore as it is built on the outdated Sheets v3 API.
If you still have code using this package, you will get this error:
```

**Review by florisvdh — APPROVED** [review]
Good move @damianooldoni :+1: , and nice result. Here's the preview:

afbeelding

File: content/tutorials/r_google_sheets/index.md [hunk 3]
```diff
@@ -1,134 +1,27 @@
...
-if (file_test("-f", "googlesheets_token.rds")) {
-  gs_auth(token = "googlesheets_token.rds")
-} else {
-  # first get the token, but not caching it in a local file
-  token <- gs_auth(cache = FALSE)
-  # save the token into a file in the current directory
-  saveRDS(token, file = "googlesheets_token.rds")
-}
-```
+## `googlesheets4` vs `googlesheet`
```
Comment (florisvdh) [inline content/tutorials/r_google_sheets/index.md:21]:
```suggestion
## `googlesheets4` vs `googlesheets`
```

File: content/tutorials/r_google_sheets/index.md [hunk 4]
```diff
@@ -1,134 +1,28 @@
...
------
+The R package [googlesheets4](https://googlesheets4.tidyverse.org/) provides the functionality to read and write data from a Google sheet.
 
-The R package [googlesheets](https://github.com/jennybc/googlesheets/blob/master/README.md) provides the functionality to retrieve data from a google sheet. Once the registration as a user is done and the permissions are granted, it enables to read and write to google sheets. Initiation of the authentification can be done with the `gs_auth` command.
+This package is very well documented, so we just have to provide you with the right links:
 
-``` r
-gs_auth()
-```
+- [read sheets](https://googlesheets4.tidyverse.org/articles/articles/read-sheets.html)
```
Comment (LienReyserhove) [inline content/tutorials/r_google_sheets/index.md:16]: I would refer to the page https://googlesheets4.tidyverse.org/articles/googlesheets4.html here instead.  The page you refer to is already more specific if I'm correct:

> Basic Sheet reading is shown in the Get started article. Here we show how to target a specific (work)sheet or cell range, how to deal with column types, and how to get detailed cell data.

File: content/tutorials/r_google_sheets/index.md [hunk 5]
```diff
@@ -1,134 +1,28 @@
...
+- [write sheets](https://googlesheets4.tidyverse.org/articles/articles/write-sheets.html)
```
Comment (LienReyserhove) [inline content/tutorials/r_google_sheets/index.md:17]: Same here, I would refer to the getting started page

**Review by LienReyserhove — COMMENTED** [review]
Great work! I added a small suggestion with respect to the link for getting started.

File: content/tutorials/r_google_sheets/index.md (hunk 4 above)
Comment (damianooldoni) [reply content/tutorials/r_google_sheets/index.md:16]: Merci @Lien! Indeed! I will change it.

Comment (damianooldoni) [reply content/tutorials/r_google_sheets/index.md]: Done: https://github.com/inbo/tutorials/pull/204/commits/cf59e8d6dccd8a9814b1df59c219163f5137fa80

File: content/tutorials/r_google_sheets/index.md (hunk 5 above)
Comment (damianooldoni) [reply content/tutorials/r_google_sheets/index.md:17]: Do you mean linking to same page in "write sheets" as well?

Comment (damianooldoni) [reply content/tutorials/r_google_sheets/index.md:17]: Yes, I see.

Comment (damianooldoni) [reply content/tutorials/r_google_sheets/index.md]: Done: https://github.com/inbo/tutorials/pull/204/commits/7462e8545f8b3f5c6628bbad3b45bd07a084eac9

**Comment by damianooldoni** [comment]
As @Lien showed, there is a getting started to point to for both writing and reading sheets. For the vast majority of the users it is more than enough. So, I link to the two chapters in the getting started. Thanks  @LienReyserhove. 
I added also the links to the other vignettes which I refer to as _specific in-depth articles_.

@florisvdh, can you have a look to it again (check my English as well :smile: ) and then merge it? Thanks.

File: content/tutorials/r_google_sheets/index.md [hunk 6]
```diff
@@ -11,16 +11,19 @@ tags: ["google", "r", "spreadsheet"]
...
 
-This package is very well documented, so we have just to provide you with the right links:
+This package is very well documented, so we just have to provide you with the right links:
 
-- [read sheets](https://googlesheets4.tidyverse.org/articles/articles/read-sheets.html)
-- [write sheets](https://googlesheets4.tidyverse.org/articles/articles/write-sheets.html)
+- [read sheets](https://googlesheets4.tidyverse.org/articles/googlesheets4.html#read_sheet-a-k-a--range_read)
+- [write sheets](https://googlesheets4.tidyverse.org/articles/googlesheets4.html#writing-sheets)
+
+There are also two specific in-depth articles about both [reading sheets](https://googlesheets4.tidyverse.org/articles/articles/read-sheets.html) and [writing sheets](https://googlesheets4.tidyverse.org/articles/articles/write-sheets.html).
```
Comment (florisvdh) [inline content/tutorials/r_google_sheets/index.md:19]:
```suggestion
There are also two specific in-depth articles: one about [reading](https://googlesheets4.tidyverse.org/articles/articles/read-sheets.html) and one about [writing](https://googlesheets4.tidyverse.org/articles/articles/write-sheets.html) sheets.
```

**Review by florisvdh — APPROVED** [review]
Fine improvements @damianooldoni @LienReyserhove ! I made a small suggestion below.

**Pull request merged**